### DISCUSSION TITLE
Additional Networks Plot X/Y LoRA

### DISCUSSION THREAD

**Discussion by Ehtz** [lllyasviel/stable-diffusion-webui-forge#281] [Q&A] [answered]
Hello,

Additional networks is not working in forge but does in A111, is there a current alternative? Thanks

https://github.com/kohya-ss/sd-webui-additional-networks

**Comment by Silvachief** [comment]
Did you have any success with this?

**Reply by Ehtz** [reply]
I am afraid not... From my understanding it uses a different Backend to Forge so it is not compatible.

**Comment by catboxanon** [comment, marked as answer]
`Prompt S/R` or this extension can be utilized: https://github.com/Yinzo/sd-webui-Lora-queue-helper

No support is planned to be given to this extension by Forge directly as the extension is deprecated by the author themself. If you wish for LoRA X/Y functionality to be improved beyond what I suggested, please open an issue upstream instead. https://github.com/AUTOMATIC1111/stable-diffusion-webui/issues/new/

**Reply by Ehtz** [reply]
I will be opening an issue upstream. Thanks for your input.
